pkgs/development/python-modules/pkg-about/default.nix

```
@@ -36,6 +36,19 @@ buildPythonPackage rec {
     tomli
   ];
 
+  # The test suite is empty, but adding the check here in case tests get added in the future
+  checkPhase = ''
+    runHook preCheck
+
+    python -m tests.__main__.py
+
+    runHook postCheck
+  '';
```

Using `doCheck` allows to skip the non-existing tests.

```suggestion
  # Module has no tests
  doCheck = false;
```